scripts/cm_test_all_sandia

```diff
@@ -1456,10 +1456,20 @@ wait_summarize_and_exit() {
       if [ $TEST_SPACK = "False" ]; then
         echo "#######################################################"
         echo "  # Reproducer instructions:"
-        cat $faildir/reload_modules.sh
-        cat $faildir/call_generate_makefile_genericpath.sh
+                  modulefile=$faildir/reload_modules.sh
+        echo "   "\#"$(cat modulefile)"
         echo ""
-        echo "  #  To reload modules, reconfigure, rebuild, and retest directly from this failing build do the following:"
+        echo "   Then run the following bash script:" 
+        echo ""
+        echo "   \\$(cat $faildir/call_generate_makefile_genericpath.sh)"
+        echo ""
+        echo "   where KOKKOSKERNELS_PATH is the top-level KokkosKernels
+        echo "   code directory. This will generate a makefile. 
+        echo ""
+        echo "   # If you have webcars Jenkins job creation access: "
+        echo ""
+        echo "  ##  To reload modules, reconfigure, rebuild, and retest "
+        echo "  ##  directly from this failing build, do the following: "
```

Comment:
@jennloe: Thank you for putting this together. Please replace your changes with the following:
```
shift
echo "    Login to $HOSTNAME and run:"
echo "      > cd /path/to/kokkos && git fetch && git checkout $KOKKOSHEADSHA"
echo "      > cd /path/to/kokkos-kernels && git fetch && git checkout $KOKKOSKERNELSHEADSHA"
echo "      > mkdir -p /path/to/kokkos-kernels-testing && cd /path/to/kokkos-kernels-testing"
echo '      > salloc -N1       # If you're on a SLURM HPC (see /opt/*_INTRO)'
echo '      > bsub -Is bash # If you're on a LSF HPC       (see /opt/*_INTRO)'
echo "      > /path/to/kokkos-kernels/scripts/cm_test_all_sandia $@"
```